packages/client/src/forms/advancedSearch/fieldDefinitions/Birth.ts

```diff
@@ -0,0 +1,394 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import { AdvancedSearchSection, IFormSectionGroup } from '@client/forms/index'
+import { formMessageDescriptors } from '@client/i18n/messages'
+import { messages as advancedSearchBirthForm } from '@client/i18n/messages/views/advancedSearchBirthForm'
+
+export const advancedSearchBirthSectionFormType: IFormSectionGroup = {
+  id: AdvancedSearchSection.Birth,
+  title: advancedSearchBirthForm.registrationDetails,
+  fields: [
+    {
+      name: 'registrationDetails',
+      type: 'ACCORDION_WITH_NESTED_FIELDS',
+      label: advancedSearchBirthForm.registrationDetails,
+      showLabel: formMessageDescriptors.showLabel,
+      hideLabel: formMessageDescriptors.hideLabel,
+      hideHeader: true,
+      required: false,
+      hideInPreview: true,
+      initialValue: 'no',
+      validate: [],
+      options: [
+        {
+          value: 'yes',
+          label: advancedSearchBirthForm.registrationDetails
+        }
+      ],
+      placeholder: advancedSearchBirthForm.registrationDetails,
+      nestedFields: {
+        yes: [
+          {
+            name: 'placeOfRegistration',
+            customisable: false,
+            type: 'LOCATION_SEARCH_INPUT',
+            label: advancedSearchBirthForm.placeOfRegistrationlabel,
+            helperText: advancedSearchBirthForm.placeOfRegistrationHelperText,
+            placeholder: formMessageDescriptors.formSelectPlaceholder,
+            required: false,
+            initialValue: '',
+            searchableResource: 'offices',
+            searchableType: 'CRVS_OFFICE',
+            dynamicOptions: {
+              resource: 'CRVS_OFFICES & ADMIN_STRUCTURE'
+            },
+
+            validate: [],
+
+            mapping: {}
+          },
+          {
+            name: 'dateOfRegistration',
+            customisable: false,
+            type: 'DATE',
+            label: advancedSearchBirthForm.dateOfRegistration,
+            required: false,
+            initialValue: '',
+            validate: []
+          },
+          {
+            name: 'statusOfRecord',
```

Comment:
@modupeadeonojobi If you take a look at your ticket, the shape of the payload is defined there. We need to keep in sync the name of the all fields with that shape. For example this `statusOfRecord` should be `registrationStatuses` instead. Could you please go through all the fields again to sync the names with the payload?